hardhat.config.ts

```diff
@@ -29,12 +31,12 @@ module.exports = {
     },
     tenderly: {
       chainId: 1,
```

Comment:
Hmm, actually the chain ID for Tenderly is not always `1`.
Probably we should make this configurable as well like `TENDERLY_CHAIN_ID`.